kernel/arch/x86_64/boot.S

```diff
@@ -183,20 +183,20 @@ hdr_start:
     .long 0x100000000 - (0xe85250d6 + 0 + (hdr_end - hdr_start))
```

Comment:
Can you also replace the 0xe85250d6 with MULTIBOOT2_HEADER_MAGIC in 180 and 183?